igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/ScmInfoController.groovy

```diff
@@ -46,6 +50,9 @@ class ScmInfoController {
 
         if(gitHubMaster)
             result << [gitHub : gitHubMaster.baseUrl]
+
+        if(bitBucketMaster)
```

Comment:
Correct, there is only a single endpoint. This was added to mimic the Stash implementation (see line 43/47).